.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,27 @@
+---
+name: "📖 Documentation"
+about: Suggest enhancement for devfile documentation
+title: ''
+labels: ''
+assignees: jc-berger
+
+---
+
+/area documentation
+
+
+**Which area this documentation change is related to?**
+
+> Uncomment appropriate `/area` lines, and delete the rest.
+> For example, `> /area api` would simply become: `/area api`
```

Comment:
Should this be enclosed in a comment to not print in the issue description? E.g. 
```suggestion
<!--
Uncomment appropriate `/area` lines, and delete the rest.
For example, `> /area api` would simply become: `/area api`
-->
```

Reply:
updated